lib/src/plugin_controller.dart

```diff
@@ -19,10 +19,10 @@ class PluginController {
     @required ArgsToProtobufConverter argsToProtobufConverter,
     @required ProtobufConverter protobufConverter,
     @required MethodChannel bleMethodChannel,
-    @required EventChannel connectedDeviceChannel,
-    @required EventChannel charUpdateChannel,
-    @required EventChannel bleDeviceScanChannel,
-    @required EventChannel bleStatusChannel,
+    @required Stream<List<int>> connectedDeviceChannel,
+    @required Stream<List<int>> charUpdateChannel,
+    @required Stream<List<int>> bleDeviceScanChannel,
+    @required Stream<List<int>> bleStatusChannel,
```

Comment:
This is a bit lazy. I'd prefer to inject _fully decoded_ streams (as we do in Hue Bluetooth), but this half-way transition is faster to implement. We can improve on it later, I guess.